lms/djangoapps/certificates/management/commands/cert_allowlist_generation.py

```diff
@@ -33,23 +35,44 @@ def add_arguments(self, parser):
             nargs="+",
             metavar='USER',
             dest='user',
-            required=True,
```

Comment:
Do we want to make the user(s) required elsewhere? Do we want to allow this to go against the whole course run?

Perhaps "yes" and this is a feature. Just wanted to raise it as a question.

Reply:
We're requiring the users down here: https://github.com/edx/edx-platform/pull/26586/files#diff-67a240fae3eff62da3f1240e56c082da4ef3931738fc03ecf950faa3d0a33c67R67-R69